mycli/main.py

```diff
@@ -841,24 +841,34 @@ def is_select(status):
         return False
     return status.split(None, 1)[0].lower() == 'select'
 
+def query_starts_with(query, prefixes):
+    """Check if the query starts with any item from *prefixes*."""
+    prefixes = [prefix.lower() for prefix in prefixes]
+    formatted_sql = sqlparse.format(query.lower(), strip_comments=True)
```

Comment:
This also makes it so `confirm_destructive_query` will work even with comments. Before, if you tried this command:
```sql
# comment
drop database foo;
```
You would not have been prompted. Now, you will.